samples/csharp_dotnetcore/81.skills-skilldialog/DialogRootBot/AdapterWithErrorHandler.cs

```diff
@@ -18,21 +18,21 @@
 
 namespace Microsoft.BotBuilderSamples.DialogRootBot
 {
-    public class AdapterWithErrorHandler : BotFrameworkHttpAdapter
+    public class AdapterWithErrorHandler : CloudAdapter
     {
         private readonly IConfiguration _configuration;
         private readonly ConversationState _conversationState;
         private readonly ILogger _logger;
-        private readonly SkillHttpClient _skillClient;
+        private readonly BotFrameworkClient _botFrameworkClient;
         private readonly SkillsConfiguration _skillsConfig;
 
-        public AdapterWithErrorHandler(IConfiguration configuration, ILogger<BotFrameworkHttpAdapter> logger, ConversationState conversationState, SkillHttpClient skillClient = null, SkillsConfiguration skillsConfig = null)
-            : base(configuration, logger)
+        public AdapterWithErrorHandler(BotFrameworkAuthentication botFrameworkAuthentication, IConfiguration configuration, ILogger<CloudAdapter> logger, ConversationState conversationState, SkillsConfiguration skillsConfig = null)
```

Comment:
```suggestion
        public AdapterWithErrorHandler(BotFrameworkAuthentication botFrameworkAuthentication, IConfiguration configuration, ILogger<IBotFrameworkHttpAdapter> logger, ConversationState conversationState, SkillsConfiguration skillsConfig = null)
```

I think we should use `IBotFrameworkHttpAdapter` instead of `CloudAdapter` so all changes are the same, and i think could be future-proof since it's an interface that any other Adapter can inherit from it

Reply:
Applied. Thanks!